3696/1/top_gems.rb

```diff
@@ -0,0 +1,57 @@
+require 'yaml'
+require 'nokogiri'
+require 'open-uri'
+require 'optparse'
+require './gem_info'
+require './options_scraper'
+require './scraper'
+# I don't think that five instance variables is too much for this case
+# Also I think that having them is better than passing useless local vars, so
+# :reek:TooManyInstanceVariables and :reek:InstanceVariableAssumption and :reek:TooManyStatements
+class TopGems
+  LARGE_NUMBER = 1_000_000
```

Comment:
better rewrite ur logic to display data for all gems if the number is not specified